### ISSUE TITLE
curl handshake failure, Ubuntu 20.04

### ISSUE THREAD

**Issue by Roleren** [ropensci/biomartr#60]
Referencing issue: https://github.com/Ensembl/ensembl-rest/issues/427

There is a bug in SSL, which now break this package on Ubuntu. 

To replicate error, start R on Ubuntu 20.04:

```
genome  <- biomartr::getGenome(db = "ensembl", "Homo sapiens",
                               path = "~/test/", gunzip = T)
```
 _Error in curl::curl_fetch_memory(rest_url) : 
  error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure_

**Comment by HajkD** [comment]
Hi @Roleren 

Thank you very much for making me aware of this new issue.

I will have a look at it and see if I can fix it or find a workaround.

Many thanks!
Hajk

**Comment by Roleren** [comment]
This error is now fixed in newest update on ubuntu 20.04, hopefully people will just update, since this is a upstream issue. 

Will close the issue.
